syft/frameworks/torch/mpc/aby3/aby3_helper.py

```diff
@@ -0,0 +1,65 @@
+import torch
+
+from syft.frameworks.torch.tensors.interpreters.replicated_shared import ReplicatedSharingTensor
+
+
+class ABY3Helper:
+    def bit_inject(rst_binary):
+        assert rst_binary.ring_size == 2
+
+        return_wrapper = rst_binary.is_wrapper
+        if return_wrapper:
+            rst_binary = rst_binary.child
+
+        players = rst_binary.players
+        shares_p1, shares_p2, shares_p3 = rst_binary.retrieve_pointers()
+        zeros = torch.zeros(shares_p1[0].shape, dtype=torch.long)
+
+        zeros_p1 = zeros.send(players[0])
```

Comment:
What else we can do here.
Using the correlated randomness generate:
(val1, val2) = gen_alpha2of3(player[0])
(val2, val3) = gen_alpha2of3(player[1])
(val3, val1) = gen_alpha2of3(player[2])

zero_share1 = val1 ^ val2
zero_share2 = val2 ^ val3
zero_share3 = val3 ^ val1
```
Where:
```zero_share1 xor zero_share2 xor zero_share3 == 0```

After that we will have:
```
x1_shares = ((x_b_1, zero_share2), (zero_share2, zero_share3), (zero_share3, x_b_1))
x2_shares = ((zero_share1, x_b_2), (x_b_2, zero_share3), (zero_share3, zero_share1))
x3_shares = ((zero_share1, zero_share2), (zero_share2, x_b_3), (x_b_3, zero_share1))
```
And at the end we will have the same result:
(x1, x2, x3 original shares in ring size2, xor can also be seen as a + here since we are in ring size 2)
Rec(x1, x2, x3) = [[x1 xor x2 xor x3]] = d + x3 - 2 * d * x3 where d = x1 + x2 - 2 * x1 * x2 (using the arithmetic shares)